src/notification.scss

```diff
@@ -29,315 +31,206 @@ $block: #{$fd-namespace}-notification;
+      display: flex;
+      align-items: baseline;
+      border-bottom: $fd-notification-border;
+      padding: 0.75rem 0.5rem 0 1rem;
 
       @include fd-rtl() {
-        padding-left: 0;
+        padding-left: 0.5rem;
         padding-right: 1rem;
       }
     }
-  }
-
-  &__close {
-    position: relative;
-    top: 0.45rem;
-    right: 0.45rem;
 
     @include fd-rtl() {
-      right: -0.45rem;
+      padding-left: 0;
+      padding-right: 0;
     }
   }
 
-  // ------------------------ BODY ------------------------
-  &__body {
-    @include fd-reset();
-
-    display: flex;
-    align-items: flex-end;
-    width: 100%;
-  }
-
   &__content {
     @include fd-reset();
 
-    display: flex;
-    padding: 1rem;
-    width: 60%;
-  }
+    width: 100%;
+    margin-left: 0.75rem;
 
-  &__avatar {
-    @include fd-reset();
+    @include fd-rtl() {
+      margin-right: 0.75rem;
+      margin-left: 0;
+    }
 
-    margin-right: 1rem;
-    min-width: 2rem;
-    min-height: 2rem;
+    &:first-child {
+      margin-left: 0;
 
-    @include fd-rtl() {
-      margin-right: 0;
-      margin-left: 1rem;
+      @include fd-rtl() {
+        margin-right: 0;
+      }
     }
-  }
 
-  &__text {
-    @include fd-reset();
+    .fd-notification__title {
+      @include fd-reset();
+      @include line-clamp(2);
 
-    width: 100%;
-    display: flex;
-    flex-direction: column;
-    padding-right: 1rem;
-  }
+      font-weight: bold;
+      font-size: var(--sapFontHeader6Size);
+    }
 
-  &__description {
-    @include fd-reset();
-    @include multiline-truncate($lineHeight: 1.45em, $lineCount: 2, $backgroundColor: $fd-notification-background-color);
+    .fd-notification__paragraph {
+      @include fd-reset();
+      @include line-clamp(2);
 
-    color: $fd-notification-content-text-color;
-  }
+      padding-top: 0.5rem;
+    }
 
-  &__metadata {
-    @include fd-reset();
+    .#{$block}__footer {
+      padding-top: 0.5rem;
+      margin: 0;
 
-    color: $fd-notification-content-text-color;
-    margin-top: 0.375rem;
+      .#{$block}__footer--separator {
+        margin: 0 0.5rem;
+      }
+    }
   }
 
-  // ------------------------ FOOTER ------------------------
-  &__footer {
+  &__header {
     @include fd-reset();
 
-    width: 40%;
     display: flex;
-    align-items: center;
-    justify-content: space-between;
-    padding: 0 0.5rem 0.5rem 1rem;
-
-    & > * {
-      &:not(:last-child) {
-        margin: 0.25rem;
-      }
-    }
   }
 
   &__actions {
     @include fd-reset();
 
-    margin-left: auto;
     display: flex;
-
-    & > * {
-      margin: 0.25rem;
-    }
+    align-items: self-start;
+    margin-left: 1rem;
 
     @include fd-rtl() {
       margin-left: 0;
+      margin-right: 1rem;
     }
-  }
-
-  &--error {
-    border-left-color: $fd-notification-indicator-border-color-error;
 
-    @include fd-rtl() {
-      border-right-color: $fd-notification-indicator-border-color-error;
+    .fd-button:first-child {
```

Comment:
Take this outside to `&__actions` and add 1 more element, which will be caled `fd-notification__button`. so it's going to be:
  ```
.#{$fd-namespace}-button.#{$block}__button {
    margin-right: 0.5rem;
   
    &::last-child { margin-right: 0 .... }
  }
```
Also let's add RTL support there